src/components/Login/Login.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import styles from './Login.module.css';
+import logo from '../../assets/logo.png';
+
+function Login() {
+	const [user, setUser] = useState('null');
+	const { register, handleSubmit, formState: { errors } } = useForm();
+
+	const onSubmit = (events) => {
+		setTimeout(() => {
+			// TODO: Consume backend endpoint
+			if (events.email === 'luz@gmail.com' && events.password === '123456') {
+				setUser(events);
+
+				window.location.href = '/dashboard';
```

Comment:
This is not the correct way to change the path on react, this is forcing the refresh of the site so you are going to lose the state.

check the react-router-dom documentation there are hooks and components that you can use for achive the correct navigation